packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/index.stories.ts

```diff
@@ -12,19 +12,23 @@ export default {
 const Template: Story<StorybookEmailArgs> = (args) => storybookEmail(args);
 
 const defaultVariables = {
-  buttonText: 'Sync another device',
+  action: 'Sync another device',
   oneClickLink: true,
-  anotherDeviceUrl:
-    '/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
+  link: 'http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
```

Comment:
Is there a reason this was renamed `link`? Since there's other links on the page, I don't know if it's clear what link this is for 🤔

Reply:
It's actually coming like that from the [`email.js`](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/email.js#L2906) file